README.md

```diff
@@ -73,6 +73,10 @@ Two environment variables can be used to alter Provsn's operation:
 - adding a root user
 - adding hashes to the blocklist
 
+### How can I get ssh access to the instances?
+
+Add you ssh-key to the list of keys available in `env.sh`, like this: https://github.com/ipfs/infrastructure/blob/master/base/env.sh#L9 and then submit a PR with the changes.
```

Comment:
There is `env.sh` is root directory too. So saying that it is `/base/env.sh` would probably be better.